conans/cli/commands/upload.py

```diff
@@ -0,0 +1,73 @@
+import json
+
+from conans.cli.api.conan_api import ConanAPIV2
+from conans.cli.api.model import UploadBundle
+from conans.cli.command import conan_command, COMMAND_GROUPS, OnceArgument
+from conans.client.userio import UserInput
+from conans.errors import ConanException
+
+
+@conan_command(group=COMMAND_GROUPS['creator'])
+def upload(conan_api: ConanAPIV2, parser, *args):
+    """
+    Uploads a recipe and binary packages to a remote.
+    By default, only the latest revisions matching the reference are uploaded, unles
```

Comment:
```suggestion
    By default, only the latest revisions matching the reference are uploaded, unless
```

Reply:
Please update the help to the actual behavior of the command and mention the latest placeholder thing